### ISSUE TITLE
inventory

### ISSUE THREAD

**Issue by fr20L** [ImpactDevelopment/ImpactIssues#3150]
inventory tweaks will be a good feature for 1.17 versions

**Comment by fr20L** [comment]
and some more survival based hacks

**Comment by phmage** [comment]
I agree on more survival oriented "utilities" but we have a lot more than we can ask for. Post more specific things on what you'd like to see. I'm curious as well. I use Baritone, X-Ray, but I play discreet and legit to remain under the radar. I also use a few others but situational. AutoArmour, AutoTool, AutoEat, etc.
